features/shared/checkout_previous_branch_after_success/deleted_current_branch/previous_branch_same.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Allow checking out previous git branch to work correctly after running a Git Town commmand that deletes the current branch
+
+  (see ../same_current_branch/previous_branch_same.feature)
+
+
+  Scenario: checkout previous branch after git-kill deletes current branch
+    Given I have feature branches named "previous" and "current"
+    And I am on the "previous" branch
+    And I switch to the "current" branch
+    And I run `git kill`
+    When I checkout my previous git branch
+    Then I end up on the "previous" branch
+
+
+  Scenario: checkout previous branch after a git-prune-branches deletes current branch
+    Given I have feature branches named "previous" and "current"
+    And the following commit exists in my repository
+      | BRANCH   | LOCATION | FILE NAME     | FILE CONTENT     |
+      | previous | local    | previous_file | previous content |
+    And I am on the "previous" branch
+    And I switch to the "current" branch
+    And I run `git prune-branches`
+    When I checkout my previous git branch
+    Then I end up on the "previous" branch
+
+
+  Scenario: checkout previous branch after git-ship deletes current branch
+    Given I have feature branches named "previous", "current"
+    And the following commit exists in my repository
+      | BRANCH  | LOCATION | FILE NAME    | FILE CONTENT    |
+      | current | remote   | feature_file | feature content |
+    And I am on the "previous" branch
+    And I switch to the "current" branch
+    And I run `git ship -m "feature done"`
+    When I checkout my previous git branch
+    Then I end up on the "previous" branch
```

Comment:
They do the same thing, but the scenarios don't necessarily have the same setup.

Off-topic: I think I'm seeing a bug in github's rendering here:
![screen shot 2015-04-23 at 10 04 48 am](https://cloud.githubusercontent.com/assets/1256911/7302759/7750a338-e9a0-11e4-8fd5-ad54d9a87ef1.png)